spark-etl/src/main/scala/geotrellis/spark/etl/EtlConf.scala

```diff
@@ -48,6 +48,8 @@ class EtlConf(args: Seq[String]) extends ScallopConf(args){
   val tileSize     = opt[Int]("tileSize",
                       descr = "Tile size in pixels (default: 256)",
                       default = Some(256))
+  val bufferSize   = opt[Int]("bufferSize",
```

Comment:
To be honest, not sure, think just preventing of its calculation depending on tiles metadata. Probably for `Etl` we can remove it, but left for the `Ingest` object.